portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+/** Adds a random fact to the page. */
+const addRandomFact = async () => {
+  const response = await fetch('/random-fact');
+  const fact = await response.text();
```

Comment:
It's good practice to wrap this in a try-catch block to handle the await call failing.